server/upload.py

```diff
@@ -0,0 +1,63 @@
+from models import *
+from sqlalchemy import update
+from datetime import datetime, timedelta
+import json
+
+
+def save_models(simulation_results, backtest_id):
+    params = simulation_results['params']
+    results = simulation_results['results']
+    profit_reset = simulation_results['profitReset']
+    profit_no_reset = simulation_results['profitNoReset']
+    ticker = simulation_results['ticker']
+
+    # update backtest
+    update(Backtest).where(Backtest.id == backtest_id).values(success=True)
```

Comment:
This update should probably not be here. We're updating the status of the backtest for each simulation, it should be one step higher.